### DISCUSSION TITLE
如何整理文件目录, 将生成的中间文件, 以及模板文件藏好, 只专注自己编辑的文件?

### DISCUSSION THREAD

**Discussion by sisrfeng** [tuna/thuthesis#785] [Q&A] [answered]
请问要改哪些变量和路径?

**Comment by Harry-Chen** [comment, marked as answer]
现在的方式已经将 tex 文件放置在 `data` 中，图片放在 `figures` 中。顶层目录无需什么改动，使用 `latexmk -c` 可以清理所有临时文件，文件也不会被 git 追踪。

**Reply by sisrfeng** [reply]
谢谢, 但感觉这会更干净:
Of these, .aux and .blg can be safely deleted. Probably .bbl too if you don't mind rerunning BibTeX and have access to the .bib files needed. The rest should not be deleted but maybe moved somewhere more appropriate. In particular .fd, .cls, and .dtx are better taking up space on a (personal) TeX tree, not in the folder of the document you're working on.

https://tex.stackexchange.com/a/7772

**Reply by xueruini** [reply]
latexmk 的 `outdir` 可以把中间文件都归拢起来，会清爽一些

**Comment by sisrfeng** [comment]
终于写完论文了, 欢迎强迫症患者使用我的配置 https://zhuanlan.zhihu.com/p/541925581

**Comment by stone-zeng** [comment]
如果用 vscode 的话，可以试一试 [file nesting](https://code.visualstudio.com/updates/v1_67#_explorer-file-nesting) 功能（当然仅仅是显示起来更清爽）。
